pkgs/shelf_router_generator/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## v1.0.4
```

Comment:
Just confirming that we plan to publish this commit+version?

Reply:
Yup!